lib/bambora/client.rb

```diff
@@ -11,26 +11,26 @@
 
 require 'bambora/client/version'
 
-# Builders
-require 'bambora/headers'
-require 'bambora/xml_request_body'
-require 'bambora/www_form_parameters'
-require 'bambora/bank/builders/payment_profile_params'
-
-# Factories
-require 'bambora/factories/response_adapter_factory'
-
 # Adapters
 require 'bambora/adapters/response'
 require 'bambora/adapters/json_response'
 require 'bambora/adapters/query_string_response'
 require 'bambora/bank/adapters/payment_profile_response'
 
+# Builders
+require 'bambora/builders/headers'
+require 'bambora/builders/xml_request_body'
+require 'bambora/builders/www_form_parameters'
+require 'bambora/bank/builders/payment_profile_params'
```

Comment:
Subtle difference here with the namespacing of the bank builders. Is the thinking that this is a separate domain, and could be moved around more easily if everything lives under `/bank`?

Reply:
Yeah, the bank stuff is a very different domain, so I chose to keep it all in that namespace. If I put it in the regular `builders` directory, the name `payment_profile_params` would easily be confused with the `v1/profiles` endpoint.